kafka-bare-quickstart/pom.xml

```diff
@@ -46,6 +46,12 @@
       <groupId>io.quarkus</groupId>
       <artifactId>quarkus-kafka-client</artifactId>
     </dependency>
+    <dependency>
+      <!-- Workaround for https://github.com/quarkusio/quarkus/issues/22000 TODO: should be removed, when quarkus/hibernate team will fix the issue -->
+      <groupId>org.apache.kafka</groupId>
+      <artifactId>kafka-clients</artifactId>
+      <version>2.8.0</version>
```

Comment:
It should be either 2.8.1 or 3.0.0, but not 2.8.0

Reply:
Changed to 2.8.1